__docs__/wonder-blocks-dropdown/single-select.stories.tsx

```diff
@@ -884,6 +888,55 @@ export const CustomOptionItems: StoryComponentType = {
     },
 };
 
+/**
+ * This example illustrates how a JSX Element can appear as the label if
+ * `labelAsText` is undefined.
+ * **Note** this is only supported for SingleSelect and MultiSelect, not Combobox.
+ */
+export const CustomOptionItemWithoutTextLabel: StoryComponentType = {
```

Comment:
Same as above. Would renaming these stories with something like "With element child as label" work for you? It would make it easier to understand them when scanning through Storybook.